_quarto.yml

```diff
@@ -15,8 +14,6 @@ website:
         text: Home
       - href: getstarted.qmd
         text: Get Started
-      - href: hub.qmd
```

Comment:
also removed this, no reason to have a separate page, should be on the landing page

Reply:
Agree!